extension-base/src/main/java/com/azure/autorest/extension/base/plugin/JavaSettings.java

```diff
@@ -137,6 +137,7 @@ public static JavaSettings getInstance() {
                 }.getType(), "polling"),
                 getBooleanValue(host, "generate-samples", false),
                 getBooleanValue(host, "generate-tests", false),
+                getBooleanValue(host, "generate-send-request-method", false),
```

Comment:
As we add more options, it's good to add a blurb on what this option is for and what gets generated when this option is enabled - both in code and in the README file.

Reply:
Agree. DPG (low-level-client option) actually have an almost fixed set of options set by default, documented at https://github.com/Azure/autorest.java#settings-for-minimal-clients-low-level-clients. And dev should not remove any of them.
Presently a few compatible option to add is also documented here.

But for vanilla, it will be really hard to choose the options just based on descriptions in README (though usually the same set of default options for DPG works reasonably OK as a starter.